src/cmd.ts

```diff
@@ -0,0 +1,32 @@
+import { Command } from 'commander'
+import { VERSION } from './version'
+
+export function parseCommandLine (argv: string[]): { source: string, target: string } {
+  const program = new Command()
+
+  program
+    .exitOverride()
+    .version(VERSION, '-v, --version')
+    .usage('[options]')
+    .option('-u, --url <url>', 'URL of the webhook proxy service. Default: https://smee.io/new')
```

Comment:
Is this "Default" value correct?  It seems like the tests in (`src/cmd.test.ts`) get `undefined` for it.

Reply:
Yes, it is the default _behavior_ but not the default _value_.  Basically, if `--url` is not provided then a new "smee.io" channel is created by a GET on https://smee.io/new.

Reply:
Ah, yes, I see...I failed to connect those dots, previously.

So, the value passed to `-u` could either be an existing channel URL (like `https://smee.io/123456`) or something which will create/redirect to one?  (That is, would/should `-u https://smee.io/new` work??  Or, does that magic only happen when `-u` is omitted?)

Reply:
> So, the value passed to `-u` could either be an existing channel URL (like `https://smee.io/123456`) or something which will create/redirect to one?

If a "source" is provided via `--url` or `-u` it must be one that is already created.

> (That is, would/should `-u https://smee.io/new` work?? Or, does that magic only happen when `-u` is omitted?)

Only when omitted.

Nothing a behavior I want to change now, but it seems a bit ... convenient.